### PR TITLE
[documentation] Added docs for prepayments

### PR DESCRIPTION
**Pull request by Lukasz-Ziemnicki**
Reference: [IN-934](https://maxioevolution.atlassian.net/browse/IN-934)

New endpoint lists prepayments for a single subscription group.

https://github.com/maxio-com/stoplight-platform-api-docs/assets/52356982/858e8770-5983-4191-bbd5-858645e3c8cf

[IN-934]: https://maxioevolution.atlassian.net/browse/IN-934?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ardahaal — APPROVED** [review]
🚀

File: reference/Chargify-API.v1.yaml [hunk 1]
```diff
@@ -19313,6 +19313,83 @@ paths:
...
+                          memo: test
+                          payment_type: cash
+                          created_at: '2023-06-21T04:37:02-04:00'
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+      operationId: get-subscription_groups-uid-prepayments.json
```
[the remark this answers is not in the document]
Comment (michalpierog) [reply reference/Chargify-API.v1.yaml]: listPrepaymentsForSubscriptionGroup - should start from lower case.
alternatively: listSubscriptionGroupPrepayments

**Review by michalpierog — CHANGES_REQUESTED** [review]
I added a few suggestion how to improve docs for code generation purposes.

**Review by michalpierog — APPROVED** [review]
LGTM

**Pull request merged**